File: vk.py
from flask import Flask, request

app = Flask(__name__)

CONFIRMATION_TOKEN = 'f52f1088'

@app.route('/callback/xE4sA', methods=['POST'])
def vk_callback():
    data = request.json
    if data.get('type') == 'confirmation':
        return CONFIRMATION_TOKEN  # Вернуть токен подтверждения
    return 'ok'

if __name__ == '__main__':
    app.run(port=80)

File: vk.php
<?php
$data = json_decode(file_get_contents('php://input')); 

$confirmation_token = 'f52f1088'; // Этот код который должен получить VK при проверке сервера ставим свои цифры
//Проверяем, что находится в поле "type" 
switch ($data->type) { 
 //Если это уведомление для подтверждения адреса сервера... 
 case 'confirmation': 
 //...отправляем строку для подтверждения адреса 
 echo $confirmation_token; 
 break;
 }
?>
